### PR TITLE
#158092345 user can create account

### PR DESCRIPTION
**Pull request by lawrenceChege**
#### What does this PR do?
Create a class for user registration
#### Description of Task to be completed?
* Create tests for User registration
* Create method for User registration using Flask-restful
* implement JWt

#### Any background context you want to provide? 
Flask-restful extention was used
flask-jwt-extended was used
#### What are the relevant pivotal tracker stories?
[#158092345](https://www.pivotaltracker.com/story/show/158092345)
#### Things to be added
connection to DB

#### Questions?
Any feedback

### CODE REVIEW

File: app/users.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# from config import conn
+
+import psycopg2
+
+
+app = Flask(__name__)
+api = Api(app)
+# cur = conn.cursor()
+app.config['JWT_SECRET_KEY'] = 'raise JSONDecodeError("Expecting value", s, err.value) from None'  # Change this!
```
Comment (victoriaaoka) [inline app/users.py]: Change what?

File: tests/test_user.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+from app import app 
+from passlib.hash import pbkdf2_sha256
+
+import unittest
+import psycopg2
+import json
+
+class TestRequestTestCase(BaseTestCase):
+    """ Test for normal user"""
+    def setUp(self):
```
Comment (victoriaaoka) [inline tests/test_user.py]: The setup is not needed here since you have it in the BaseTestCase class.

**Review by victoriaaoka — CHANGES_REQUESTED** [review]

File: tests/test_user.py (hunk 2 above)
Comment (lawrenceChege) [reply tests/test_user.py]: okay i removed it

File: app/users.py (hunk 1 above)
Comment (lawrenceChege) [reply app/users.py]: thanks  I removed it

**Pull request merged**